packages/fds-components/src/components/interactive/Toaster.jsx

```diff
@@ -0,0 +1,58 @@
+import React, { useState, useEffect } from 'react';
+import ReactDOM from 'react-dom';
+import { CSSTransition } from 'react-transition-group';
+import PropTypes from 'prop-types';
+
+/**
+ * @module Toaster
+ * Controller for showing, dismissing and positioning Toast components.
+ */
+const Toaster = ({ toast }) => {
+  const [isToasting, setIsToasting] = useState(Boolean(toast));
+  let clonedToast = null;
+
+  const dismissToast = () => {
+    setIsToasting(false);
+  };
+
+  // pass dismiss function down to toast
+  if (toast) {
+    clonedToast = React.cloneElement(toast, { dismissToast });
+  }
+
+  // set a timeout for toast to dismiss itself
+  // eslint-disable-next-line consistent-return
+  useEffect(() => {
+    if (toast && toast.props.isAutoDismiss) {
```

Comment:
May want to also check for toast.props.dismissDelay

Reply:
Toast sets `dismissDelay` as a defaultProp, so in theory it should always be there. This brings up a good point though; if `Toaster` relies on it being there, I guess I should also mark `dismissDelay` as a required prop in `Toast`?

Reply:
oh cool! yeah didn't see that, should be fine

Reply:
on second thought, i'm going to leave it as a default prop - users shouldn't be required to pass it every time